### PR TITLE
List Item UI Updates

### PR DESCRIPTION
**Pull request by ttannis**
### Summary of Changes

This PR covers a set of changes to our list item UIs:

1. Do not display dashboard group indigo in `DashboardListItem` -- `dashboard-group` css updated accordingly.
2. Vertically centered the name/description on list items -- if there is no descriptions the name will still be centered.
3. Fixed `column-name` font css according to designs.
4. Fixed `QueryList` item height according to designs.
5. `ChartList` needed a `key` on each mapped item to avoid console warnings
6. Update `TabsComponent` css to have a border under the tabs. Usages were also updated to avoid a double-thick line at the top if the first list item is directly under the tabs.
7. Do not show `ChartList` if there are no charts.

### Tests

Tests updated according to changes.

### Documentation

N/A

### CheckList
Make sure you have checked **all** steps below to ensure a timely review.
- [x] PR title addresses the issue accurately and concisely. Example: "Updates the version of Flask to v1.0.2"
    - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
- [x] PR includes a summary of changes, including screenshots of any UI changes. 
- [x] PR adds unit tests, updates existing unit tests, __OR__ documents why no test additions or modifications are needed.
- [x] In case of new functionality, my PR adds documentation that describes how to use it.
    - All the public python functions and the classes in the PR contain docstrings that explain what it does
- [x] PR passes all tests documented in the [developer guide](https://github.com/lyft/amundsenfrontendlibrary/blob/master/docs/developer_guide.md#testing)

### CODE REVIEW

**Comment by ttannis** [comment]
@danwom PR history is weird from branching off a branch but the file diff for review is correct.

**Comment by ttannis** [comment]
@danwom let me know if you have any comments on this one

File: amundsen_application/static/css/styles.scss [hunk 1]
```diff
@@ -51,3 +51,8 @@ form {
   overflow: hidden;
   text-overflow: ellipsis;
 }
+
+.my-auto {
```
Comment (danwom) [inline amundsen_application/static/css/styles.scss:55]: Curious of the name choice. What does `my` stand for? `mg-auto` or `margin-auto` might make more sense.

Edit: Oh I get it... margin-y-auto.

Comment (ttannis) [reply amundsen_application/static/css/styles.scss:55]: Yep.

**Pull request merged**